src/index.css

```diff
@@ -174,3 +174,54 @@ code {
   padding-left: 1.5rem;
   padding-right: 1.5rem;
 }
+
+.w-fit {
+  width: fit-content;
+}
+
+.shareButtonContainer {
```

Comment:
instead of creating custom CSS classes in this index.css file, we're trying to use [styled-components](https://styled-components.com/). Let me know if you want to grab time to discuss how to implement that, but you should be able to just define a `ShareButtonContainer` component at the top of ShareButton like:
```
const ShareButtonContainer = styled.div`
  position: absolute;
  right: 0;
  top: 47%;
  transform: translate(0%, -50%);
`
```
and then use it as a normal react component

Reply:
Done 👍